src/osmet-pack

```diff
@@ -48,14 +48,15 @@ set -x
 
 # we want /dev/disk symlinks for coreos-installer
 /usr/lib/systemd/systemd-udevd --daemon
-/usr/sbin/udevadm trigger --settle
+/usr/sbin/udevadm trigger -v --settle --type=subsystems --action=add
+/usr/sbin/udevadm trigger -v --settle --type=device --action=add
```

Comment:
I was seeing this hang sometimes when running locally.

Reply:
Which part?